fe/fe-core/src/test/java/com/starrocks/analysis/ShowRoutineLoadStmtTest.java

```diff
@@ -0,0 +1,55 @@
+// This file is made available under Elastic License 2.0.
+// This file is based on code available under the Apache license here:
+//   https://github.com/apache/incubator-doris/blob/master/fe/fe-core/src/test/java/org/apache/doris/analysis/ShowTableStmtTest.java
+
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package com.starrocks.analysis;
+
+import com.starrocks.qe.ConnectContext;
+import com.starrocks.sql.analyzer.SemanticException;
+import com.starrocks.utframe.UtFrameUtils;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ShowRoutineLoadStmtTest {
+
+    private ConnectContext ctx;
+
+    @Before
+    public void setUp() throws Exception {
+    }
+
+    @Test
+    public void testNormal() throws Exception {
+        ctx = UtFrameUtils.createDefaultCtx();
+        ctx.setCluster("default_cluster");
+        ctx.setDatabase("testDb");
+
+        ShowRoutineLoadStmt stmt = new ShowRoutineLoadStmt(new LabelName("testDb","label"), false);
```

Comment:
Please generate the Stmt with the Parser, which could test the parser logic.

Reply:
I can generate the PauseRoutineLoadStmt with Parser
please tell how to genertate the PauseRoutineLoadStmtTest with Parser
thank you

Reply:
Use `com.starrocks.sql.parser.SqlParser.parse()` could parse a SQL string and got a statement.